### PR TITLE
MODCAL-68 update dropsColumns.sql, add removeDefauldIds, setNotNullCo…

### PR DESCRIPTION
**Pull request by roman-barannyk**
## Purpose
- Create sql script to remove using default id value;
- Create sql script to set NOT NULL constraint for jsonb columns;
- Include all UUID functions;

Resolves: [MODCAL-68](https://issues.folio.org/browse/MODCAL-68)

## Approach
- update schema.json;
- create SQL script;

## Links
https://issues.folio.org/browse/MODCAL-68

### CODE REVIEW

File: src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+DO $$ BEGIN ALTER TABLE exceptional_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE openings ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE regular_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE exceptions ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+
+DO $$ BEGIN ALTER TABLE audit_actual_opening_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE audit_exceptional_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE audit_openings ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE audit_regular_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE audit_exceptions ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
```
Comment (julianladisch) [inline src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql:11]: The audit_* tables changed to a new incompatible jsonb schema: https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-26
Either delete/truncate the existing content and mention it in the release notes, or convert the content of each jsonb field to the new format.

File: src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+DO $$ BEGIN ALTER TABLE actual_opening_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE exceptional_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE openings ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE regular_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE exceptions ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
```
Comment (julianladisch) [inline src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql:5]: The `creation_date` needs to get converted from `timestamp with time zone` to `timestamp without time zone`.

File: src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql (hunk 1 above)
Comment (roman-barannyk) [reply src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql:11]: If I delete the existing content, do you think it is safety?

File: src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql (hunk 2 above)
Comment (roman-barannyk) [reply src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql:5]: updated

Comment (julianladisch) [reply src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql:5]: Thanks, resolved.

File: src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql (hunk 1 above)
Comment (roman-barannyk) [reply src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql:11]: Thanks, fixed

**Comment by alexanderkurash** [comment]
We're going to merge this in order to prepare a bugfix release. @julianladisch if you have any concerns we can address them in the follow-up pull request.

**Pull request merged**